wdl-lint/src/rules/shellcheck.rs

```diff
@@ -396,6 +477,11 @@ fn to_bash_var(placeholder: &Placeholder, ty: Option<Type>) -> (String, bool) {
                     true,
                 );
             }
+            PrimitiveType::String => {
```

Comment:
the docstring for the `to_bash_var` function needs to be updated to reflect these changes. And the name of the function should probably also change